docs/pr-workflow.md

```diff
@@ -7,109 +7,60 @@ Let's say you've been assigned to work on a bug or story... where do you branch
 Since we mentioned we work from two forks of the repo, it is useful to understand the main branches we typically use to develop on this project.  A majority of Flexion developers will work off of the Flexion fork, but we often need to get work merged into the Court's repo.  Take note of the following branches:
 
 ### Flexion Fork Branches
-- `origin/develop` (for **passive** branch development)
-- `origin/staging` (for **active** branch development)
-- `origin/experimental1` (for deploying to a test environment)
-- `origin/experimental2` (for deploying to a test environment)
-- `origin/experimental3` (for deploying to a test environment)
-- `origin/experimental4` (for deploying to a test environment)
-- `origin/experimental5` (for deploying to a test environment)
 
-We will discuss what **passive** and **active** means later in this documentation, but for the most part you will work from develop or staging when dealing with refactoring or story work, and occasionally push your changes to an experimental branch to verify if major devops or AWS work is required.
+- `flexion/develop` (for demonstrating complete work to the P.O.)
+- `flexion/experimental1` (for deploying to a test environment)
+- `flexion/experimental2` (for deploying to a test environment)
+- `flexion/experimental3` (for deploying to a test environment)
+- `flexion/experimental4` (for deploying to a test environment)
+- `flexion/experimental5` (for deploying to a test environment)
 
-Multiple times during a sprint we need to interact with the US Tax Court fork.  Take note of the following branches:
+You will occasionally push your changes to an experimental branch to verify if major devops or AWS work is required.
 
 ### US Tax Court Fork Branches
-- `upstream/migration` (a staging environment for the PO to verify story work)
-- `upstream/test` (kept up to date with `prod`, but a staging environment for PO to verify bug fixes)
-- `upstream/staging` (kept up to date with `prod`, but also a staging environment for the upcoming deploy)
-- `upstream/prod` (the current version which is running on production)
 
-If you are at Flexion, you will never need to touch `upstream/prod`.
+Multiple times during a sprint we need to interact with the US Tax Court fork.  Take note of the following branches.
 
-?> The court often commits code to their staging branch which requires us to back merge changes into our Flexion fork.  See the [Environment Shuffle](/pr-workflow.md?id=environment-shuffle) section below for more information.
+- `court/hotfix` (a staging environment for when we need to fix something in production without also releasing other story/bug work)
```

Comment:
is the plan for the court to create a hotfix environment?  or do we just plan to merge the `hotfix` code into mig to get it deployed / tested?

Reply:
The idea was that we would rename `court/migration` to `court/hotfix` at some point.